src/fragments/lib/auth/js/enable-signin-signup-signout.mdx

```diff
@@ -30,37 +30,77 @@ User attributes are not required by default. You can make user attributes requir
 
 </Callout>
 
-#### How will you verify users at sign-up?
+### How will you verify users at sign-up?
 
 Once the initial sign-up is done, the user will need to verify and confirm their contact information before their account is activated. This is done by sending a verification code via email or phone number that can be collected on initial sign-up. Email verification asks the user to click a link or enter a code sent to their email address provided during sign-up. Phone verification is done by asking the user to enter the code sent to them in a SMS text message. You can also enable auto-confirmation to skip this step and automatically confirm the user after sign-up. This is accomplished using the `PreSignUp` Lambda trigger which is explained in the [Amazon Cognito documentation](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-pre-sign-up.html).
  
-#### Do you need local or global sign-out?
+### Do you need local or global sign-out?
 
 The default when passing the `Auth.signOut()` method is a local sign out. This will sign-out the user from a specific application or device and revokes the tokens used for authentication in that session. If you need a global sign-out that revokes all associated tokens, credentials, and sessions across all applications and devices, then you update the method to `Auth.signOut({ global: true })`.
 
+Once you have these decisions in mind you are ready to enable sign-up, sign-in, and sign-out.
+
 ## Add the sign-up, sign-in, and sign-out capabilities
 
 You can add the functionality to enable sign-up, sign-in, and sign-out with the Amplify Libraries. You can connect this to an existing UI or create your own.
```

Comment:
What else, if any, do we want to bring over from this page for the below sections? https://docs.amplify.aws/lib/auth/emailpassword/q/platform/js/ There are some additional explainers on here - not sure if they will be moved to another area of the TOC or if they could be in expanders etc. I actually like how simplified this page is to follow so I'm not sure we want to add them without an expander but should depend on if we are missing anything they have to do to be successful.

Reply:
I think we have enough in this new page. Which explainers specifically do you think we should move?

Reply:
The auto sign in and custom attributes are added.